wildfire/lib/screens/all_updates/all_updates.dart

```diff
@@ -0,0 +1,352 @@
+                                "1h ago",
+                                style: GoogleFonts.openSans(
+                                  textStyle: TextStyle(
+                                    color: Color(0xff797474),
+                                    fontSize: 10.sp,
+                                  ),
+                                ),
+                              ),
+                            ],
+                          ),
+                          SizedBox(height: 4.h),
+                          SizedBox(
+                            width: 157.w,
+                            height: 140.h,
+                            child: Text(
+                              "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Tincidunt dignissim sagittis sociis tincidunt metus faucibus quis. Amet et mattis mattis fames. Velit.",
+                              style: GoogleFonts.openSans(
+                                textStyle: TextStyle(
+                                  color: Color(0xff333333),
+                                  fontSize: 13.sp,
+                                ),
+                              ),
+                            ),
+                          ),
+                        ],
+                      ),
+                    ),
+                  ],
+                ),
+              ),
+              Column(children: [
+                SizedBox(
+                  height: 16.h,
+                ),
+                SizedBox(
+                  width: 343.w,
+                  height: 21.h,
+                ),
+                SizedBox(height: 8.h),
+                Container(
+                  width: 343.w,
+                  height: 240.h,
+                  decoration: BoxDecoration(
+                    borderRadius: BorderRadius.circular(24.r),
+                    color: Color(0xffebebeb),
+                  ),
+                  child: Column(
+                    children: [
+                      Row(
+                        mainAxisSize: MainAxisSize.min,
+                        mainAxisAlignment: MainAxisAlignment.center,
+                        crossAxisAlignment: CrossAxisAlignment.center,
+                        children: [
+                          StreamBuilder<QuerySnapshot>(
+                              stream: FirebaseFirestore.instance
+                                  .collection('fire-updates')
+                                  .snapshots(),
+                              builder: (BuildContext,
+                                  AsyncSnapshot<QuerySnapshot> querySnapshot) {
+                                if (querySnapshot.hasData) {
+                                  return Center(
+                                    child: CircularProgressIndicator(),
+                                  );
+                                } else
+                                  return ListView.builder(
+                                      itemBuilder: (context, index) {
```

Comment:
Are you sure you properly pushed your code because the red line that was here because of not returning widget it required at the right is still and you fixed it initially.